packages/desktop-electron/index.ts

```diff
@@ -246,11 +247,11 @@ app.on('ready', async () => {
 
     if (pathname.startsWith('/static')) {
       callback({
-        path: path.normalize(`${__dirname}/client-build${pathname}`),
+        path: path.normalize(`${__dirname}/../client-build${pathname}`),
```

Comment:
Entrypoint is the dist folder now - we need to go back one directory to get to the "external" code